asn1ate/sema.py

```diff
@@ -327,23 +331,23 @@ def resolve_tag_implicity(self, tag_implicity, tagged_type_decl):
         # Tagged CHOICEs must always be explicit if the default is implicit, automatic or empty
         # See X.680, 30.6c
         if isinstance(tagged_type_decl, ChoiceType):
-            return TagImplicity.EXPLICIT
+            return TagImplicitness.EXPLICIT
 
         # No tag implicity specified, use module-default
         if self.tag_default is None:
             # Explicit is default if nothing
-            return TagImplicity.EXPLICIT
-        elif self.tag_default == TagImplicity.AUTOMATIC:
+            return TagImplicitness.EXPLICIT
+        elif self.tag_default == TagImplicitness.AUTOMATIC:
             # TODO: Expand according to rules for automatic tagging.
-            return TagImplicity.IMPLICIT
+            return TagImplicitness.IMPLICIT
 
         return self.tag_default
 
     def __str__(self):
         return '%s DEFINITIONS ::=\n' % self.name \
```

Comment:
I wonder if this expression would be simpler if you used trailing `+` instead of line continuation. Should have the same effect, I think.